content/docs/replication/_index.md

```diff
@@ -31,8 +31,8 @@ CSM for Replication provides the following capabilities:
 {{<table "table table-striped table-bordered table-sm">}}
 | COP/OS        | PowerMax         | PowerStore       | PowerScale |
 |---------------|------------------|------------------|------------|
-| Kubernetes    | 1.23, 1.24, 1.25 | 1.22, 1.23, 1.24 | 1.22, 1.23, 1.24 |
-| Red Hat OpenShift | 4.10, 4.11    | 4.9, 4.10        | 4.9, 4.10  |
+| Kubernetes    | 1.22, 1.23, 1.24 | 1.22, 1.23, 1.24 | 1.22, 1.23, 1.24 |
```

Comment:
The change was correct. I think the table talks about Powermax supporting replication on 1.25 cluster.